package.json

```diff
@@ -1,7 +1,10 @@
 {
-  "name": "poet-node",
-  "version": "1.0.0",
+  "name": "@po.et/node",
+  "version": "1.0.1",
```

Comment:
How are we keeping this up-to-date? If I go to npm, the latest version is 1.0.3, not 1.0.1.

Reply:
Sorry! Forgot to include the last commit. Please check now